sample/ml_sample_validator.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
I suggest renaming the schema to: `ml_sample_schema.json` similar to the original one (sample_schema.json). Calling it validator suggests this is the validator which it is not.

**Note**: Remember to update the `$id` value.

Reply:
Done